Program.cs

```diff
@@ -0,0 +1,22 @@
+﻿using System;
+
+namespace Task2
+{
+    class MainClass
+    {
+        public static void Main(string[] args)
+        {
+            int a, b, C, c, P;
+            double S;
```

Comment:
Локальные переменные принято называть с маленьких букв. Имена лучше давать значимые, чтобы, читая код, было понятно, что означает та или иная величина.
```suggestion
            int sideA, sideB, angleC, sideC, perimeter;
            double area;
```